201-vm-efficientip-vhd/azuredeploy.json

```diff
@@ -48,12 +48,6 @@
         "description": "Name of the existing VNET"
       }
     },
-    "existingVirtualNetworkResourceGroup": {
```

Comment:
this is still needed in the subnet ref - should be the first parameter of the resourceId function

Reply:
Added back existingVirtualNetworkResourceGroup parameter and updated at subnetref